docsets/JUCE/docset.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "JUCE",
-    "version": "5.4.7",
-    "archive": "JUCE.tgz",
+    "version": "6.0.0",
+    "archive": "versions/6.0.0/JUCE.tgz",
```

Comment:
This doesn't work. The main archive needs to be at a different location from the ones in `specific_versions`. I'll fix it.

Reply:
thanks! wasn't sure if it'd work or not, now i know 👍